src/PullRequest/Domain/Aggregate/PullRequest/PullRequestDescription.php

```diff
@@ -8,7 +8,7 @@
 
 class PullRequestDescription
 {
-    public const TARGET_BRANCH_AVAILABLE = ['develop', '8.1.x'];
+    public const TARGET_BRANCH_AVAILABLE = ['develop', '8.2.x', '8.2.x'];
```

Comment:
are you sure to duplicate `8.2.x`?

Reply:
NO. This is what I get for trying to fix CI when tired at 10pm 😅